model.rb

```diff
@@ -162,6 +162,11 @@ def deposit(amount)
     @balance += amount
   end
 
+  # Prints current account balance
+  def show_balance 
```

Comment:
You should delete this method as this exact method already exists as ```balance``` which was create by ```attr_accessor```. Read about it here if you're not sure what I mean: https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby